aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -571,6 +571,100 @@ Tensor& cholesky_out(Tensor &result, const Tensor &self, bool upper) {
   return result;
 }
 
+template<typename scalar_t>
+static void  apply_cholesky_mod(const Tensor& self, Tensor & out, Tensor & err, bool upper, std::vector<int64_t>& infos) {
+#ifndef USE_LAPACK
+  AT_ERROR("cholesky: LAPACK library not found in compilation");
+#else
+  char uplo = upper ? 'U' : 'L';
+
+  Tensor outTag =  out;
```

Comment:
This assignment can be done in the else clause of the branch `batch_size > 1` (line 590).